### PR TITLE
Patch release of alpha.25, to merge in so that it's on master

### PR DESCRIPTION
**Pull request by carver**
bump py-evm, for eth-keys dependency patch

### CODE REVIEW

**Comment by carver** [comment]
Crucially, the docker-image-build-test passed, which is what tipped me off to the dependency issue in the first place.

**Comment by carver** [comment]
So this branch includes the tagged commit that got released. I think the right thing to do is to merge it back into master, which should have pretty minor effects. One bonus is that it bumps the version and folds in the py-evm upgrade. The only "wacky" thing it does is mean that the next version released will not be all based on the previous version. But I think it's better than the alternatives.

Any opposition to merging this in? @pipermerriam @cburgdorf @lithp @ralexstokes

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import os
 from setuptools import setup, find_packages
 
-PYEVM_DEPENDENCY = "py-evm==0.2.0a43"
+PYEVM_DEPENDENCY = "py-evm==0.3.0a1"
```
Comment (cburgdorf) [inline setup.py]: I think we should cherry-pick that into `master` if we don't cut a new release today.

Comment (cburgdorf) [reply setup.py]: Oh I just read your comment that you also want it to be merged. Let's do it!

Comment (carver) [reply setup.py]: Sweet, I'll wait a few hours to see if anyone has objections.

Comment (carver) [reply setup.py]: Actually, since it's holding up #586 I'll probably merge it as soon as I finish clearing my github queue.

**Pull request merged**